components/script/dom/window.rs

```diff
@@ -969,7 +969,8 @@ impl Window {
             return
         }
 
-        self.force_reflow(goal, query_type, reason)
+        self.force_reflow(goal, query_type, reason);
+        assert!(!self.Document().needs_reflow() || self.window_size.get().is_none());
```

Comment:
Can you add a comment indicating why we need to weaken the assert case in the window_size.get().is_none() case?